cmd/updater/diffdumps/cmd.go

```diff
@@ -99,10 +107,10 @@ func generateK8sDiffs(outputDir string, baseZipR *zip.ReadCloser, headZipR *zip.
 		}
 
 		// Only look at YAML files in the k8s/ folder.
-		if filepath.Dir(name) != vulndump.K8sDirName || filepath.Ext(name) != ".yaml" {
+		if filepath.Dir(name) != dirName || filepath.Ext(name) != ".yaml" {
 			continue
 		}
-		if err := generateK8sDiff(k8sSubDir, baseFiles[name], headF); err != nil {
+		if err := generateK8sDiff(subDir, baseFiles[name], headF); err != nil {
```

Comment:
It should not be generateK8sDiff here. This function does not seem to be generic to different component.

Reply:
Yeah, this is wrong. I will fix this part.